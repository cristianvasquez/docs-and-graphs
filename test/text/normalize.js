import { expect } from 'expect'

import toMatchSnapshot from 'expect-mocha-snapshot'
import {
  normalizeText,
} from '../../src/text/normalize.js'

const text = [
  'An #autumn scene showing a beautiful #horse coming to visit me.',
  '#nowplaying Pointer Sisters - Dare Me | #80s #disco #funk #radio',
  '- A list item',
  '## Header',
  '   Trim this  '
]


expect.extend({ toMatchSnapshot })

describe('normalizeText', async function () {
  for (const current of text) {
    it(current, async function () {
      const result = normalizeText(current)
      expect(result).toMatchSnapshot(this)
    })
  }
})
