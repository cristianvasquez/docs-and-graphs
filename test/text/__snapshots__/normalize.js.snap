// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`normalizeText    Trim this   1`] = `"Trim this"`;

exports[`normalizeText ## Header 1`] = `"Header"`;

exports[`normalizeText #nowplaying Pointer Sisters - Dare Me | #80s #disco #funk #radio 1`] = `"Pointer Sisters - Dare Me |"`;

exports[`normalizeText - A list item 1`] = `"A list item"`;

exports[`normalizeText An #autumn scene showing a beautiful #horse coming to visit me. 1`] = `"An  scene showing a beautiful  coming to visit me."`;
